aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -0,0 +1,186 @@
+Deploy an Ubuntu Pro EKS cluster
+================================
+
+This tutorial shows how to deploy an EKS Cluster using Ubuntu Pro node and FIPS enabled. If you don't need FIPS, you can also use cloud-init with the ubuntu-pro-advantage module.
+
+Why using Packer
+~~~~~~~~~~~~~~~~
+We use Packer because to enable FIPS, you need to reboot the machine. We use Packer to create a golden AMI to avoid flagging the node as defective.
+ 
+What other approaches are there
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you use Ubuntu Pro service without needing to reboot the image, use:
+
+- Cloud-init see `here <https://cloudinit.readthedocs.io/en/latest/reference/modules.html#ubuntu-advantage>`_
```

Comment:
this could use an example. You can paste the cloud-init configuration, and tell readers to skip the Packer section if not using FIPS.

Later when deploying the cluster, you can offer 2 eksctl instructions: with and without cloud-init